external_modules/src/peanut_moveit_interface/srv/MoveitInterface.srv

```diff
@@ -0,0 +1,3 @@
+geometry_msgs/Pose goal_pose
+---
```

Comment:
should this also have something to indicate the grip?

Reply:
as per our discussion, we'll have a separate interface for the gripper as it is a separate component